pykafka/managedbalancedconsumer.py

```diff
@@ -0,0 +1,368 @@
+        :type offsets_commit_max_retries: int
+        :param auto_offset_reset: What to do if an offset is out of range. This
+            setting indicates how to reset the consumer's internal offset
+            counter when an `OffsetOutOfRangeError` is encountered.
+        :type auto_offset_reset: :class:`pykafka.common.OffsetType`
+        :param consumer_timeout_ms: Amount of time (in milliseconds) the
+            consumer may spend without messages available for consumption
+            before returning None.
+        :type consumer_timeout_ms: int
+        :param rebalance_max_retries: The number of times the rebalance should
+            retry before raising an error.
+        :type rebalance_max_retries: int
+        :param rebalance_backoff_ms: Backoff time (in milliseconds) between
+            retries during rebalance.
+        :type rebalance_backoff_ms: int
+        :param auto_start: Whether the consumer should start after __init__ is complete.
+            If false, it can be started with `start()`.
+        :type auto_start: bool
+        :param reset_offset_on_start: Whether the consumer should reset its
+            internal offset counter to `self._auto_offset_reset` and commit that
+            offset immediately upon starting up
+        :type reset_offset_on_start: bool
+        :param post_rebalance_callback: A function to be called when a rebalance is
+            in progress. This function should accept three arguments: the
+            :class:`pykafka.balancedconsumer.BalancedConsumer` instance that just
+            completed its rebalance, a dict of partitions that it owned before the
+            rebalance, and a dict of partitions it owns after the rebalance. These dicts
+            map partition ids to the most recently known offsets for those partitions.
+            This function can optionally return a dictionary mapping partition ids to
+            offsets. If it does, the consumer will reset its offsets to the supplied
+            values before continuing consumption.
+            Note that the BalancedConsumer is in a poorly defined state at
+            the time this callback runs, so that accessing its properties
+            (such as `held_offsets` or `partitions`) might yield confusing
+            results.  Instead, the callback should really rely on the
+            provided partition-id dicts, which are well-defined.
+        :type post_rebalance_callback: function
+        :param use_rdkafka: Use librdkafka-backed consumer if available
+        :type use_rdkafka: bool
+        :param compacted_topic: Set to read from a compacted topic. Forces
+            consumer to use less stringent message ordering logic because compacted
+            topics do not provide offsets in stict incrementing order.
+        :type compacted_topic: bool
+        :param heartbeat_interval_ms: The amount of time in milliseconds to wait between
+            heartbeat requests
+        :type heartbeat_interval_ms: int
```

Comment:
This appears to be copied over from the base class. Ideally, we don't repeat details. There may be decent tools out there for this, but in a pinch, you could grab `__doc__` from the super and merge in the local docstring. This could e.g. be done in a decorator.